theme/less/main.less

```diff
@@ -48,3 +48,34 @@ main
   padding: 15px;
   max-width: 1000px;
 }
+
+#breadcrumb
+{
+  font-size: 0.92em;
+  color: #999;
+  background-color: #f1f1f1;
+  border-radius: 4px;
+  border: 1px solid #e9e9e9;
+  text-shadow: 1px 1px 0px #fff;
+}
+
+#breadcrumb a
+{
+  margin-right: 6px;
```

Comment:
In general consistent left and right margins and/or paddings simplify implementation, especially when it comes to supporting RTL(Right to left) languages it plays huge role. Even though we don't plan supporting RTL at this point, when it's possible to have consistent left and right distances would be great to use that opportunity.

In this case I can see that consistent distances are being created by `#breadcrumb a` and `.parts:after` so instead of this maybe we can just have?
```
.parts:after {
  margin: 0px 6px;
}
```

That way presumably we can just get rid of current rule.